torch/utils/data/dataloader.py

```diff
@@ -139,6 +139,9 @@ class DataLoader(Generic[T_co]):
                  See `Dataset Types`_ for more details on these two types of datasets and how
                  :class:`~torch.utils.data.IterableDataset` interacts with
                  `Multi-process data loading`_.
+
+    .. warning:: See :ref:`reproducibility`, and :ref:`dataloader-workers-random-seed`, and
```

Comment:
This seems more like a note:: than a warning:: to me.

It's great to link these docs here. Can this be more descriptive than "random seed related questions?" Maybe something like:

"See the x, y, and z notes for information about reproducible data loading."?